tests/test_unified_scoring.py

```
@@ -0,0 +1,234 @@
+                generation_prompt="test prompt"
+            ),
+            GeneratedIdea(
+                content="Idea 2: Wind turbines",
+                thinking_method="deduction",
+                agent_name="test_agent",
+                generation_prompt="test prompt"
+            )
+        ]
+        
+        # Create config
+        config = EvolutionConfig(
```

Ruff (0.12.2)

Local variable `config` is assigned to but never used

Remove assignment to unused variable `config`

(F841)